src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/PlacementRequestTransformerTest.kt

```diff
@@ -322,4 +324,41 @@ class PlacementRequestTransformerTest {
       ),
     )
   }
+
+  @ParameterizedTest
+  @EnumSource(value = ReleaseTypeOption::class)
```

Comment:
so test scenario loops for every release type, right? Nice.

Reply:
correct. i'm not sure why we're not enumerating these values but suspect that could open a can of worms so this at least gives us safety going forward.